### PR TITLE
R refactor

### PR DESCRIPTION
**Pull request by tesmonrd**
Established SQL, and got it to insert new scores on game completion, also integrated jQuery into game.js
Also assignment took 7hrs; it was fun applying what we've been learning in class on our blogs on a new project with different code

### CODE REVIEW

**Comment by tesmonrd** [comment]
Also assignment took 7hrs; it was fun applying what we've been learning in class on our blogs on a new project with different code

**Comment by mirandalily** [comment]
Assignment took 2 lab sessions to complete. The first day we put in about 3.5 hours, the second day around 5. It went pretty smoothly and actually helped a lot with my understanding of SQL.

File: js/webdb.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    console.error('Error occured during DB init. Web Database may not be supported.');
+  }
+};
+
+webDB.connect = function (database, title, size) {
+  html5sql.openDatabase(database, title, size);
+};
+
+webDB.setupTables = function () {
+  $.get('sql/setup-tables.SQL',function(sqlStatements){
```
Comment (joelstanner) [inline js/webdb.js:40]: File not found - it's not in this location.

File: setup-tables.SQL [hunk 2]
```diff
@@ -0,0 +1 @@
+CREATE TABLE IF NOT EXISTS articles (id INTEGER PRIMARY KEY, title VARCHAR(255) NOT NULL, author VARCHAR(255) NOT NULL, authorUrl VARCHAR (255), category VARCHAR(20), publishedOn DATETIME, body TEXT NOT NULL);
```
Comment (jonathanstallings) [inline setup-tables.SQL:1]: It looks like you are not using this file.

File: js/scores.js [hunk 3]
```diff
@@ -20,6 +22,46 @@ function makeTable(){
   }
 }
 
+var createSQL = function(callback) {
+  webDB.execute(
+    [
+      {
+        'sql': 'CREATE TABLE IF NOT EXISTS highscores ( id INT, n TEXT, s INT, PRIMARY KEY (id));',
```
Comment (jonathanstallings) [inline js/scores.js:29]: Your `id` column is not set up correctly for WebSQL; the id column remains blank for all inserted records. Also, try to use more semantic names for your database columns.

File: js/game.js [hunk 4]
```diff
@@ -133,6 +134,18 @@ var Shuffler = {
   }
 };
 
+var newInsertSQL = function(callback) {
```
Comment (jonathanstallings) [inline js/game.js:137]: Where is this function being called?

File: js/game.js [hunk 5]
```diff
@@ -231,19 +245,23 @@ var Responder = {
...
+              'sql': 'INSERT INTO highscores (n, s) VALUES (?, ?);',
+              'data' : [highScores[i].n, highScores[i].s],
+            },
+          ]);
         window.location = 'scores.html';
       }
-    });
+    }
   }
 };
```
Comment (jonathanstallings) [inline js/game.js:267]: Below are a few more places you could use jQuery for event listeners.

File: js/game.js [hunk 6]
```diff
@@ -1,3 +1,4 @@
+webDB.init();
 var spotOne = document.getElementById('spotOne');
```
Comment (jonathanstallings) [inline js/game.js:2]: This could be done with jQuery.

**Pull request merged**